bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -802,9 +809,16 @@ func (s *ksServer) SaveAppToRepo(project string, email string, repoDir string) e
 	if err != nil {
 		return err
 	}
-
-	return runCmd(fmt.Sprintf("git config user.email '%s'; git config user.name 'auto-commit'; git add .; "+
-		"git commit -m 'auto commit from deployment'; git pull --rebase; git push origin master", email))
+	if err = runCmd(fmt.Sprintf("git config user.email '%s'; git config user.name 'auto-commit'", email)); err != nil {
+		return err
+	}
+	if err = runCmd("git add .; git commit -m 'auto commit from deployment'"); err != nil {
+		return err
+	}
+	if err = runCmd("git pull --rebase; git push origin master"); err != nil {
```

Comment:
I'm not really sure what/where we're pushing here, but if we're doing this do we want to do a force push?

Reply:
We are use cloud source repo.
force push might not be a good idea, just in case user have changes pushed around same time.